### DISCUSSION TITLE
Back-to-back notifications that my personal access token has expired?

### DISCUSSION THREAD

**Discussion by dougscott** [community/community#70038] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I got this email notice on 9/18/23:

----------------------
_Hi @dougscott,
We noticed your personal access token (classic) "Source Tree Access" with gist, repo, user, and workflow scopes will expire in 6 days.
If this token is still needed, visit https://github.com/settings/tokens/9590616/regenerate to generate an equivalent.
If you run into problems, please contact support by visiting https://github.com/contact?tags=dotcom-accounts
Thanks,
The GitHub Team_
----------------------

That same day I updated it, and got a confirmation titled "[GitHub] A personal access token (classic) has been regenerated for your account".

Yesterday I got the following email:

----------------------
_Hi @dougscott,
We noticed your personal access token (classic) "Source Tree Access" with gist, repo, user, and workflow scopes will expire in 7 days.
If this token is still needed, visit https://github.com/settings/tokens/9590616/regenerate to generate an equivalent.
If you run into problems, please contact support by visiting https://github.com/contact?tags=dotcom-accounts
Thanks,
The GitHub Team_
----------------------

I don't understand - I only have one account with one token - why is this happening?

Thank you for any help

**Comment by yavuzakyazici** [comment]
Got it today.

Hey @yavuzakyazici,

Your personal access token (classic) "Xcode IDE" with audit_log, codespace, copilot, delete:packages, delete_repo, notifications, project, repo, workflow, and write:packages scopes has expired.

If this token is still needed, visit https://github.com/settings/tokens/1380788589/regenerate to generate an equivalent.

If you run into problems, please contact support by visiting https://github.com/contact?tags=dotcom-accounts

Thanks,
The GitHub Team
